RELEASE_NOTES.md

```diff
@@ -20,6 +20,8 @@ A new header is inserted each time a *tag* is created.
 - gltfio: UbershaderProvider now takes the ubershader archive in its constructor [⚠️ **API Change**]
 - gltfio: Fix morphing with sparse accessors.
 - gltfio: Fix models that use signed integers for morphing.
+- gltfio: Fix adreno gpu crash introduced from gpu morph target change
```

Comment:
Please put the release note under the `main branch` heading. Also, we can stick with a single release note with the `engine` tag.

Reply:
Hi, @bejado , thanks for the review and the recommendation.
I moved the note to main branch, and since now there's only change in engine, I removed gltfio line.